stock_orderpoint_manual_procurement/models/stock_warehouse_orderpoint.py

```diff
@@ -41,6 +41,13 @@ def _get_procure_recommended_qty(self, virtual_qty, op_qtys):
 
     @api.depends("product_min_qty", "product_id", "qty_multiple")
     def _compute_procure_recommended(self):
+        # '_quantity_in_progress' override in 'purchase_stock' method has not
+        # been designed to work with NewIds (resulting in KeyError exceptions).
+        # To circumvent this, we knowingly skip such records here.
+        for op in self:
```

Comment:
@sebalix Usage of update() is better in this case.

```suggestion
        self.update({"procure_recommended_qty": False})
```

Reply:
Thank you, updated